### ISSUE TITLE
Indicate folders / devices where the remote end is paused.

### ISSUE THREAD

**Issue by acolomb** [syncthing/syncthing#8310]
If a remote device has accepted a folder we share to them, but set it to paused state, the only indication right now is the device completion never reaching 100 percent. This should be marked on a more granular basis for each shared folder in the device details / each "Shared With" device in the folder details.

The info is already available by looking at missing folder entries from the protocol's ClusterConfig message and exposed through the interface added in #8283.  It should be indicated in the GUI just like for folders not shared back from the remote, with a footnote number.

**Comment by acolomb** [comment]
Seems this was omitted from the v1.20.2-rc.1 release notes?

**Comment by calmh** [comment]
Yeah because the commit message didn't mention this issue so it wasn't picked up. Fixing manually.

**Comment by uok** [comment]
@acolomb, this is a good idea, but the small superscript numbers from https://github.com/syncthing/syncthing/pull/8286 are hard to spot. 
Suggestions: (colored) pause icon, different color, strike through text for paused device

**Comment by acolomb** [comment]
There was some discussion about this both on the forum and in the respective PR. The footnote numbers seemed to be the best consensus.

I think a more general kind of troubleshooting interface would be a good solution which I've been pondering for some time. It could summarize many such deficiencies, e.g.
* pending devices and folders
* not accepted / paused remote folders
* possibly not accepted remote device connections if we have a reason from them why the connection is being closed
* suggested additional devices that already share a folder indirectly through other remotes
* out of sync files
* conflicted files
* etc. 

Basically anything you currently need to notice in a lot of different places, but with better explanation and ideally hints on how to fix it. If you're into helping with the design of such a thing, I guess we should discuss and design it on the forum first. Same goes for small visual changes to the current indicators.

**Comment by uok** [comment]
I understand your idea which could be a bigger improvement later down the road
see forum https://forum.syncthing.net/t/mark-folders-which-are-not-accepted-by-remote/18116 for reference
